src/bin/mcp.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+import { Command } from "../command";
+import { requireAuth } from "../requireAuth";
+
+import { silenceStdout } from "../logger";
+silenceStdout();
+
+import { FirebaseMcpServer } from "../mcp/index";
+
+const cmd = new Command("mcp").before(requireAuth);
+
+export async function mcp() {
+  const options: any = {};
```

Comment:
No action required. mostly just a note to self:
cmd.prepare is probably in need of a refactor to be strongly typed and return Promise<Options> instead of mutating the input.